### DISCUSSION TITLE
Getting Unable to resolve model with family selection: gpt-4

### DISCUSSION THREAD

**Discussion by pedro-rccl** [community/community#128137] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Getting Unable to resolve model with family selection: gpt-4

**Comment by Akash1134** [comment]
Hi @pedro-rccl ,

Please check that Copilot Chat is enabled in Settings. (Copilot --> Policies) (Switch to the Organization page instead of personal GitHub account, if you're part of one)

![image](https://github.com/community/community/assets/95834130/cdd40887-900c-498a-b14b-89e425828033)

Thanks!
